### DISCUSSION TITLE
Polish translation updated

### DISCUSSION THREAD

**Discussion by hskrok** [aclap-dev/video-downloadhelper#1366] [Language & Translation]
[pl.txt](https://github.com/user-attachments/files/17402323/pl.txt)

**Comment by lrusinowski** [comment]
i tried to mimic the way english translation has been written
[pl.txt](https://github.com/user-attachments/files/17508408/pl.txt)

**Comment by PNiedbalski** [comment]
[pl.txt](https://github.com/user-attachments/files/17953562/pl.txt)

**Comment by EnterRandomString** [comment]
Hi. I have updated, fixed and polished the Polish translation.
[pl.txt](https://github.com/user-attachments/files/18034951/pl.txt)

**Comment by Andba215** [comment]
Hi, I filled in some empty fields that I found in the translation file

[pl.txt](https://github.com/user-attachments/files/18079819/pl.txt)

**Comment by Aviquo** [comment]
[pl.txt](https://github.com/user-attachments/files/18293310/pl.txt)

**Comment by CloudFluff** [comment]
[pl.txt](https://github.com/user-attachments/files/18551073/pl.txt)

**Comment by gelu22** [comment]
[pl.txt](https://github.com/user-attachments/files/18587813/pl.txt)
